docs/privacy_policy.md

```diff
@@ -0,0 +1,54 @@
+## Privacy Policy
+
+Georepublic UG (haftungsbeschränkt) built the Consento app as an Open Source app. This SERVICE is provided by Georepublic UG (haftungsbeschränkt) at no cost and is intended for use as is.
```

Comment:
Just for the record: I am okay with Georepublic being used. @marcchataigner is surely fine with it too.

Reply:
The template gives two options: individuals or organisations. 
I don't care really, but I also do not know the impllications ... in the end it's about responsibilites probably and so I thought it's better to write a company name.

Reply:
SERVICE: Does this need to be capitalized? I'm just curious.